ngraph/frontend/onnx/onnx_import/src/op/max_pool.cpp

```diff
@@ -23,15 +25,23 @@ namespace ngraph
                     {
                         NGRAPH_WARN << "MaxPool: Indices output is not supported and was ignored";
                     }
-                    auto max_pool = pooling::PoolingFactory(node).make_max_pool();
+                    auto max_pool = PoolingFactory(node).make_max_pool();
                     max_pool.emplace_back(std::make_shared<NullNode>()); // Indices (optional)
                     return max_pool;
```

Comment:
Since this is an optional trailing output can we remove this NullNode from here?
I think it was added to cheat indices output when it was required.